pkg/plugin/manager/manager.go

```diff
@@ -90,6 +102,38 @@ func (m *manager) RunMigrations() error {
 	return nil
 }
 
+func (m *manager) runPublicMigrations() error {
+	// Setup DB if not set
+	if m.db == nil {
+		setErr := m.setDB()
+		if setErr != nil {
+			return errors.New(fmt.Sprintf("could not open db: %s", setErr.Error()))
```

Comment:
conversely, would use `fmt.Errorf` here and format the error with `%w` instead of `%s` (and then you don't need to call `.Error()` on the error you're formatting)

Reply:
True - this came over via a little known dev technique of copy and paste.